pygmt/base_plotting.py

```diff
@@ -606,47 +606,52 @@ def grdview(self, grid, **kwargs):
 
     @fmt_docstring
     @use_alias(
-        R="region",
-        J="projection",
+        A="straight_lines",
         B="frame",
-        S="style",
+        C="cmap",
+        D="offset",
+        E="error_bars",
+        F="connection",
         G="color",
+        I="intensity",
+        J="projection",
+        L="close",
         N="no_clip",
-        W="pen",
-        i="columns",
-        l="label",
-        C="cmap",
+        R="region",
+        S="style",
         U="timestamp",
         V="verbose",
+        W="pen",
         X="xshift",
         Y="yshift",
+        Z="zvalue",
+        i="columns",
```

Comment:
Have we used the *columns* alias for **-i** elsewhere? It may be a bad choice, because GMT also has **-o** which controls columns of output.

In https://github.com/GenericMappingTools/gmt/pull/230, GMT uses *read-columns* for **-i** and *write-columns* for **-o**.

GMT.jl uses *incol* for **-i** and *outcol* for **-o**.

Reply:
It is used in `contour`:

https://github.com/GenericMappingTools/pygmt/blob/v0.2.0/pygmt/base_plotting.py#L576

I can take this out though, since it's not as useful in the Python world where we can select columns using indexing.

Reply:
Yes I agree.

Reply:
Oops, actually i="columns" has already been aliased before in plot, just that the code has changed position. So I think we should keep it :sweat_smile:

Reply:
OK, let's break backward compatibility later.